my_profiles/example/builds.yaml

```diff
@@ -11,6 +11,12 @@
 
 # In this example, we use these default methods. See other templates for examples of how to customize this subsampling scheme.
 
+# Define input files.
+inputs:
+  - name: example-data
+    metadata: data/example_metadata.tsv
+    sequences: data/example_sequences.fasta.gz
```

Comment:
This currently fails, as the compressed fasta is handed to `nextalign` which results in an empty `results/aligned_example-data.fasta` (no error raised). The pipeline eventually fails at a downstream filter step (which is slightly confusing).

We could add a check to `_get_path_for_input` which throws a nice error if the input looks like it's compressed.

Reply:
e.g. 

```diff
diff --git a/workflow/snakemake_rules/common.smk b/workflow/snakemake_rules/common.smk
index d4a09568..105d16b2 100644
--- a/workflow/snakemake_rules/common.smk
+++ b/workflow/snakemake_rules/common.smk
@@ -53,6 +53,10 @@ def _get_path_for_input(stage, origin_wildcard):
     if path_or_url=="" and stage in ["metadata", "sequences"]:
         raise Exception(f"ERROR: config->input->{origin_wildcard}->{stage} is not defined.")
 
+    ## Currently only metadata can be compressed!
+    if stage!="metadata" and (path_or_url.endswith(".xz") or path_or_url.endswith(".gz")):
+        raise Exception(f"Compressed sequence input is not yet supported. Please check {path_or_url}")
+
     if stage=="metadata":
         return f"data/downloaded_{origin_wildcard}.tsv" if remote else path_or_url
     if stage=="sequences":
```

Reply:
(And we should update `my_profiles/example/builds.yaml` to use `data/example_sequences.fasta`. The docs already instruct you to decompress the sequences file before running.)

Reply:
Good catch! It turns out that the example profile has been broken on `master` for a while now (since the proximity script started ignoring the reference strains), so we'll need to fix that in a separate PR.

In the meantime, I switched the starting sequences for the example profile back to the uncompressed file. This compressed vs uncompressed issue should be mostly handled by #605. I was hoping to merge that PR first, so this PR won't need to add any checks for compression state.